dds/DCPS/SequenceNumber.h

```diff
@@ -39,7 +39,7 @@ class OpenDDS_Dcps_Export SequenceNumber {
         // this code is here, despite the RTPS spec statement:
         // "sequence numbers never wrap"
         this->high_ = 0;
-        this->low_ = 1;
+        this->low_ = 0;
```

Comment:
Why is 0 considered part of the normal range of sequence numbers?  Could this be left at 1?

Reply:
I'm not sure why this code is here at all if the spec says it's not supposed to wrap. Maybe just have ++ do nothing when sequence number is at max value?